src/main/java/frc/robot/Robot.java

```diff
@@ -332,14 +303,17 @@ public void robotInit() {
         .and(operator.rightBumper())
         .toggleOnFalse(
             Commands.parallel(
-                elevator.setExtensionCmd(() -> ElevatorSubsystem.CLIMB_EXTENSION_METERS),
+                climber.extendClimbCmd(),
```

Comment:
i think elevator needs to extend for trap while we are on the ground, either here or when y is first pressed